pkg/util/admissioncheck/admissioncheck_test.go

```diff
@@ -34,18 +32,7 @@ import (
 )
 
 func TestConfigHelper(t *testing.T) {
-	testConfig := &kueue.ProvisioningRequestConfig{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "config",
-		},
-		Spec: kueue.ProvisioningRequestConfigSpec{
-			ProvisioningClassName: "className",
-			Parameters: map[string]kueue.Parameter{
-				"p1": "v1",
-			},
-			ManagedResources: []corev1.ResourceName{"cpu"},
-		},
-	}
+	testConfig := utiltesting.MakeProvisioningRequestConfig("config").ProvisioningClass("className").WithParameter("p1", "v1").WithManagedResource("cpu").Obj()
```

Comment:
```suggestion
	testConfig := utiltesting.MakeProvisioningRequestConfig("config").
		ProvisioningClass("className").
		WithParameter("p1", "v1").
		WithManagedResource("cpu")
```

nit: just for readability